server/index.js

```diff
@@ -110,3 +110,4 @@ async function onError(error) {
 }
 
 module.exports = { start };
+require('make-runnable');
```

Comment:
What is that? Why do we need it?

Reply:
I added this library: https://www.npmjs.com/package/make-runnable so we can run this as a script in the command line: 
```
> node index start
```